instrumentation/jvm-metrics/src/main/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/JvmMetricsInstaller.java

```diff
@@ -59,8 +57,25 @@ public void afterAgent(AutoConfiguredOpenTelemetrySdk autoConfiguredOpenTelemetr
       new JvmThreadMetrics().bindTo(Metrics.globalRegistry);
     }
     if (useOtelMetrics(metricsImplementation)) {
-      new OtelJvmGcMetrics().install();
-      new OtelJvmHeapPressureMetrics().install();
+      // gc metrics were removed:
+      //   runtime.jvm.gc.concurrent.phase.time is replaced by OTel
+      //     process.runtime.jvm.gc.duration{gc=<concurrent gcs>}
+      //   runtime.jvm.gc.pause is replaced by OTel
+      //     process.runtime.jvm.gc.duration{gc!=<concurrent gcs>}
+      //   runtime.jvm.gc.max.data.size is replaced by OTel
+      //     process.runtime.jvm.memory.limit{pool=<long lived pools>}
+      //   runtime.jvm.gc.live.data.size is replaced by OTel
```

Comment:
The `runtime.jvm.gc.pause` and `runtime.jvm.gc.live.data.size` metrics are used by Splunk Memory Profiling, we should not remove them from the agent until there is full server-side support for the new metrics.